prac_05/state_names.py

```diff
@@ -0,0 +1,24 @@
+STATE_NAMES = {
+    "QLD": "Queensland",
+    "NSW": "New South Wales",
+    "VIC": "Victoria",
+    "TAS": "Tasmania",
+    "SA": "South Australia",
+    "WA": "Western Australia",
+    "NT": "Northern Territory",
+    "ACT": "Australian Capital Territory"
+}
+
+
+def main():
+    short_state = input("Enter state abbreviation: ").upper()
+
```

Comment:
STATE_NAMES dictionary. If the abbreviation exists in the dictionary, its corresponding full state name is returned; otherwise, None is returned